src/Process/SubdivisionControl.js

```diff
@@ -1,12 +1,33 @@
+import { STRATEGY_GROUP, getGroupLevel } from '../Core/Layer/LayerUpdateStrategy';
+
 export default {
     preUpdate: (context, layer) => {
         context.colorLayers = context.view.getLayers(
             (l, a) => a && a.id == layer.id && l.type == 'color');
         context.elevationLayers = context.view.getLayers(
             (l, a) => a && a.id == layer.id && l.type == 'elevation');
-    },
 
+        context.allGroupElevationLevels = [];
+        for (const e of context.elevationLayers) {
+            if (e.updateStrategy.type == STRATEGY_GROUP) {
+                context.allGroupElevationLevels = context.allGroupElevationLevels.concat(e.updateStrategy.options.groups);
+            }
+        }
+        context.allGroupElevationLevels.sort((a, b) => a - b);
+    },
     hasEnoughTexturesToSubdivide: (context, layer, node) => {
+        // Prevent to subdivise the node if the current elevation level
+        // is below the target level in group strategy.
+        // we must avoid a tile, with level 20, inherits a level 3 elevation texture.
+        // The induced geometric error is much too large and distorts the SSE
```

Comment:
I fail to see why this is specific to the `STRATEGY_GROUP`?
Basically you want to prevent subdivision if `node.level - elevationLevel` is too big.

And IMHO, the bug should be fixed in the SSE logic (https://github.com/iTowns/itowns/blob/master/src/Process/LayeredMaterialNodeProcessing.js#L60 is another place that got patched for the same SSE logic issue).

Reply:
> I fail to see why this is specific to the STRATEGY_GROUP?

indeed, we can extend this logic to all strategies.

> And IMHO, the bug should be fixed in the SSE logic (https://github.com/iTowns/itowns/blob/master/src/Process/LayeredMaterialNodeProcessing.js#L60 is another place that got patched for the same SSE logic issue).

On the other hand, if this logic is placed at this point, the tile will be displayed first without any elevation texture, it will be a regression..